packages/pn-pa-webapp/src/components/NewNotification/Recipient.tsx

```diff
@@ -146,52 +135,31 @@ const Recipient: React.FC<Props> = ({
         .test('taxIdDependingOnRecipientType', t('fiscal-code-error'), function (value) {
           return taxIdDependingOnRecipientType(value, this.parent.recipientType);
         }),
-      digitalDomicile: yup.string().when('showDigitalDomicile', {
-        is: true,
-        then: requiredStringFieldValidation(tc, 320).matches(dataRegex.email, t('pec-error')),
```

Comment:
why did you remove the requiredStringFieldValidation?

Reply:
Because now "digitalDomicile" is an input always displayed but not required